cylc/flow/task_pool.py

```diff
@@ -712,37 +754,17 @@ def _release_tasks(self):
             itask.waiting_on_job_prep = True
             self.data_store_mgr.delta_task_state(itask)
             self.data_store_mgr.delta_task_queued(itask)
-        if released:
-            LOG.debug(
-                "Queue released:\n"
-                + '\n'.join(f"* {r.identity}" for r in released)
-            )
```

Comment:
This involves less iteration, since we're already iterating over the list for the preceding lines. Do you have a strong preference either for debug vs info on this? (It's debatable IMO).